std_msgs/Header header
uint32 id
uint32 seq
uint32 lv_gps
float64 latitude
float64 longitude
float64 altitude
geometry_msgs/Quaternion quaternion